File: BackEnd/src/domain/interfaces/usecases/community/IUnlikePostUseCase.ts
/**
 * @file IUnlikePostUseCase.ts
 * @description Interfaz del caso de uso para quitar el like de un post.
 * @module Community
 * @layer Domain
 */
export interface IUnlikePostUseCase {
  execute(postId: string, userId: string): Promise<void>;
}

File: FrontEnd/lib/domain/interfaces/usecases/community/i_unlike_post_use_case.dart
/// @file i_unlike_post_use_case.dart
/// @description Interfaz: Quita el like de un post.
/// @module Community
/// @layer Domain
library;
abstract interface class IUnlikePostUseCase {
  /// Quita el like de un post.
  Future<void> execute(String postId);
}
